templates/security/cve/_cve-footer.html

```diff
@@ -0,0 +1,46 @@
+<section class="p-section">
```

Comment:
This file was already reviewed and the only change is that I moved it into a partial to make the base page a bit more readable, so no need to review this one